LLM/OSS/service.py

```
@@ -67,19 +68,62 @@ def init_db_pool() -> None:
+            )
+            _ssh_tunnel.start()
+            tunnel_db_kwargs = dict(db_kwargs, host="localhost", port=_ssh_tunnel.local_bind_port)
+            _db_pool = pg_pool.ThreadedConnectionPool(minconn=1, maxconn=5, **tunnel_db_kwargs)
+            logger.info(
+                "chatbot_db_pool_initialized ssh_tunnel=%s db_host=%s db_port=%s",
+                True,
+                tunnel_db_kwargs["host"],
+                tunnel_db_kwargs["port"],
+            )
+            return
+        except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)